internal/journal/voljournal.go

```diff
@@ -581,6 +580,9 @@ func (conn *Connection) ReserveName(ctx context.Context,
 		nameKeyVal = volUUID
 	}
 
+	// After generating the UUID Directory omap, we populate the csiDirectory
+	// omap with a key-value entry to map the request to the backend volume:
+	/// `csiNameKeyPrefix + reqName: nameKeyVal`
```

Comment:
`///` to `//` or use multiline comment?

Reply:
Thanks for catching the typo! Updated :+1: